src/store/reducers/queryResults.js

```diff
@@ -137,6 +132,7 @@ export function matchNewPostIntoQueryResults (state, { id, isPublic, type, group
     // Chat posts only appear in the chat rooms, nowhere else
     if (type !== 'chat') {
       queriesToMatch.push(
+        // TODO: add types here
```

Comment:
??

Reply:
oh thats an old comment ive been meaning to work on. unrelated. but would be handled by the new data stuff